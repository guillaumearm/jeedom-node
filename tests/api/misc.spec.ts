import { prepareJeedomTest } from '../utils'

describe('rpc.misc', () => {
  const [api, recordTest] = prepareJeedomTest()

  recordTest('ping', async () => {
    expect(await api.ping()).toMatchSnapshot()
  })
  recordTest('version', async () => {
    expect(await api.version()).toMatchSnapshot()
  })
  recordTest('datetime', async () => {
    expect(await api.datetime()).toMatchSnapshot()
  })
})
