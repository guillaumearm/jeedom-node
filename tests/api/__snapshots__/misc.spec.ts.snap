// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rpc.misc datetime 1`] = `1541448131.8188`;

exports[`rpc.misc ping 1`] = `"pong"`;

exports[`rpc.misc version 1`] = `"3.2.11"`;
